CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Adds support for building OCI images with [podman](https://podman.io/), e.g. `operator-sdk build --image-builder=podman`. ([#1488](https://github.com/operator-framework/operator-sdk/pull/1488))
 - New option for [`operator-sdk up local --enable-delve`](https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#up), which can be used to start the operator in remote debug mode with the [delve](https://github.com/go-delve/delve) debugger listening on port 2345. ([#1422](https://github.com/operator-framework/operator-sdk/pull/1422))
 - Enables controller-runtime metrics in Helm operator projects. ([#1482](https://github.com/operator-framework/operator-sdk/pull/1482))
+- Generating and serving info metrics about each custom resource. By default these metrics are exposed on port 8686. ([#1277](https://github.com/operator-framework/operator-sdk/pull/1277))
```

Comment:
@LiliC We'll need probably need a user doc to explain what the "info metrics" on each custom resource are, i.e the CR count, and guide how to add other operator-specific metrics.

The [proposal](https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/metering-operator-metrics.md) should suffice for now but we should create an issue to track the docs for this PR after this is merged. Unless you want to track that in the umbrella issue #616

Reply:
I have a PR out for this already :) https://github.com/operator-framework/operator-sdk/pull/1521